module/applications/compendium-browser.mjs

```diff
@@ -200,7 +250,10 @@ export default class CompendiumBrowser extends HandlebarsApplicationMixin(Applic
   async _renderResults() {
     let rendered = [];
     for ( const entry of await this.#results ) {
-      const context = { entry };
+      const context = {
+        img: entry.img ?? game.dnd5e.moduleArt.map.get(entry.uuid.replace("Actor.", ""))?.actor,
```

Comment:
Yeah, the fresh indexes are missing the art since we only apply it on first index fetch. It would probably be cleaner to wrap `CompendiumCollection#getIndex` to ensure this is done every time, but we can't do that at the moment without monkey patching so this seems easier (but I've got a core issue for it: https://github.com/foundryvtt/foundryvtt/issues/10228).

Reply:
Actually, this seems to cause an issue if you load the compendium browser for actors and then open the actors compendium because none of the art is present. I think I'll need to make a public method on `ModuleArt` that will re-apply the art to a newly-generated index.